ibm_catalog.json

```diff
@@ -528,39 +572,46 @@
               "service_name": "containers-kubernetes",
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::serviceRole:Manager",
-                "crn:v1:bluemix:public:iam::::role:Editor"
+                "crn:v1:bluemix:public:iam::::role:Administrator"
               ],
-              "notes": "Required to create and edit the OpenShift cluster and the related resources."
+              "notes": "Required to reset API keys, create and edit the OpenShift cluster, and manage all related resources."
             },
             {
               "service_name": "iam-identity",
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Administrator",
                 "crn:v1:bluemix:public:iam-identity::::serviceRole:UserApiKeyCreator"
               ],
-              "notes": "Required to create the containers-kubernetes-key needed by the OpenShift cluster on IBM Cloud."
+              "notes": "Required to create the containers-kubernetes-key needed by the OpenShift cluster on IBM Cloud and for managing and operating resources within the IBM Cloud environment."
             },
             {
               "service_name": "is.vpc",
               "role_crns": [
-                "crn:v1:bluemix:public:iam::::role:Administrator"
+                "crn:v1:bluemix:public:iam::::role:Editor"
               ],
               "notes": "Required for creating Virtual Private Cloud(VPC)."
             },
             {
-              "service_name": "cloud-object-storage",
+                "service_name": "cloud-object-storage",
+                "role_crns": [
+                    "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                    "crn:v1:bluemix:public:iam::::role:Editor"
+                ],
+                "notes": "Required to edit data and manage the OpenShift cluster's internal registry storage bucket."
+            },
+            {
               "role_crns": [
-                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
-                "crn:v1:bluemix:public:iam::::role:Editor"
+                "crn:v1:bluemix:public:iam::::role:Viewer"
               ],
-              "notes": "Required to create Cloud Object Storage (COS) instance."
+              "service_name": "Resource group only",
+              "notes":"Viewer access is required in the resource group you want to provision in."
             }
           ],
           "architecture": {
             "features": [
               {
                 "title": " ",
-                "description": "Configured for getting started with OpenShift AI, offering limited customization options."
+                "description": "Configures a QuickStart Deployable Architecture on IBM Cloud that enables single-click deployment of a GPU-enabled Red Hat OpenShift cluster with the OpenShift AI add-on."
```

Comment:
I remember the guideline was to use a generic statement here, you can check the parent epic. It is mentioned there and same for all DAs.